api/app/schemas/bookings/booking_schema.py

```diff
@@ -32,6 +33,8 @@ class Meta:
     room_id = fields.Int()
     start_time = fields.DateTime()
     invigilator_id = fields.Int()
+    office_id = fields.Int()
 
     invigilator = fields.Nested(InvigilatorSchema())
     room = fields.Nested(RoomSchema(exclude=("booking", "office",)))
+    office = fields.Nested(OfficeSchema())
```

Comment:
Does ExamSchema contain a nested office in it as well? And/or does office contain nested bookings in it's schema? Want to make sure that we aren't including multiple things in here.

Reply:
<img width="618" alt="screen shot 2019-01-28 at 9 51 42 am" src="https://user-images.githubusercontent.com/29988052/51855542-61c89780-22e2-11e9-8154-3a436a79b8de.png">

Reply:
<img width="471" alt="screen shot 2019-01-28 at 9 52 50 am" src="https://user-images.githubusercontent.com/29988052/51855608-80c72980-22e2-11e9-85bf-86a8928189b7.png">

Reply:
<img width="860" alt="screen shot 2019-01-28 at 9 55 57 am" src="https://user-images.githubusercontent.com/29988052/51855824-034fe900-22e3-11e9-8845-61ba41bd1ca2.png">